src/css/global/layout.scss

```diff
@@ -0,0 +1,11 @@
+// Override Protocol's 25% / 75% layout to be 33% / 67%
+
+@media #{$mq-md} {
+  .mzp-l-sidebar {
+    width: 33.333% !important;
+  }
+
+  .mzp-l-main {
```

Comment:
Looks like that is meant for this container:
https://github.com/mdn/developer-portal/pull/1285/files#diff-b7ede9403e1fba8bb7a9047a07b700e1R15

Could we add a custom class and in so doing avoid the use of `!important`, for example:


```
<div class="mzp-l-main custom-width">
...

.mzp-l-main.custom-width {
   width: 66.665%;
}
```

Reply:
Sure we can do that. I'll add it in on a future pass